core/src/main/java/de/danoeh/antennapod/core/UpdateManager.java

```diff
@@ -71,6 +71,11 @@ private static void onUpgrade(final int oldVersionCode, final int newVersionCode
                 UserPreferences.setEpisodeCleanupValue(oldValueInDays * 24);
             } // else 0 or special negative values, no change needed
         }
+        if (oldVersionCode < 1070197) {
+            if (prefs.getBoolean(UserPreferences.PREF_MOBILE_UPDATE_OLD, false)) {
```

Comment:
couldn't we just read the boolean as a string (I would assume that this return "true" or "false"), migrate if to its new value and keep using the same pref key?

Reply:
Looks risky to me. Android stores preferences in an XML file like this: `<boolean name="prefMobileUpdate" value="true" />`

Reply:
Ok, I agree.
I like the idea to hardcode the old pref key in the upgrade thingy. Keeps the prefs a lil cleaner.